tasks/package-apt.yml

```diff
@@ -8,7 +8,4 @@
     cache_valid_time: "{{ package_cache_valid_time }}"
     install_recommends: "{{ item.apt_install_recommends | default(omit) }}"
   when: (item.apt_ignore|default('no'))|string in 'False,false,No,no'
-  with_flattened:
-    - "{{ package_list }}"
-    - "{{ package_list_host }}"
-    - "{{ package_list_group }}"
+  with_items: "{{ (package_list + package_list_group + package_list_host ) | list | unique }}"
```

Comment:
Can we move that combination to `variables/main.yml` and then have a single variable in all the separate files?

eg;
```
# variables/main.yml
packages: "{{ (package_list + package_list_group + package_list_host ) | list | unique }}"
```
and then in the separate task files;
```
...
  with_items: "{{ packages }}"
```

That way we keep the logic in a single place.

Thx!

Reply:
Gertjan, updated and added vars/main.yml as requested.